### ISSUE TITLE
week 52: settle budgets onchain

### ISSUE THREAD

**Issue by sajanrajdev** [Badger-Finance/badger-multisig#1418]
Prev #1325

**Comment by sajanrajdev** [comment]
Top-up for Dec Q4 2023. Once the Budget is approved at the beginning of Q1 2024 there will be a need to further settle it on-chain.

**Comment by sajanrajdev** [comment]
Closing in favor of #1491 

Moving forward, association funding will be assessed and settled on a quarterly basis via a TCD. A ticket describing the operations required will be created based on this TCD in an as per need basis. No need to create one in advance.
